submit/submit

```diff
@@ -0,0 +1,534 @@
+#!/usr/bin/env python3
+
+#
+# submit -- command-line submit program for solutions.
+#
+# Part of the DOMjudge Programming Contest Jury System and licensed
+# under the GNU GPL. See README and COPYING for details.
+#
+
+import argparse, sys, logging, os, stat, requests, json, time
+try:
+    import magic
+except ModuleNotFoundError:
+    # Ignore, magic is optional
+    magic = None
+
+# By default no base URL is set and it is expected to use either the
+# SUBMITBASEURL environment variable or the -u/--url argument.
+# If you want to hardcode a default base URL, set it here
```

Comment:
Reword as:

# Set the default base URL to submit to (optional). It can be overridden by the
# SUBMITBASEURL environment variable or the -u/--url argument.

Reply:
Love the rendering of this comment, but intended was:
```python
# Set the default base URL to submit to (optional). It can be overridden by the
# SUBMITBASEURL environment variable or the -u/--url argument.
```